packages/ckeditor5-dev-utils/lib/bundler/geteditorconfig.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Transforms specified configuration to a string which can be inserted
+ * to generated an entry file.
+ *
+ * Keys and values will be wrapped in apostrophes instead of quotation marks.
+ *
+ * Keys which contains letters or/and numbers or/and underscore or/and a Dollar sign
+ * will not be wrapped in apostrophes.
+ *
+ * @param {Object} config
+ * @returns {String}
+ */
+module.exports = function getEditorConfig( config ) {
+	if ( !config ) {
+		return '{}';
+	}
+
+	return JSON.stringify( config, null, '\t' )
+		.split( '\n' )
+		.map( ( line, index ) => {
+			if ( index === 0 ) {
+				return line;
+			}
+
+			return `\t${ line }`;
+		} )
+		.join( '\n' )
+		// Replace quotation marks to apostrophes.
+		.replace( /"/g, '\'' )
```

Comment:
This is potentially breaking. You can have `"\""` string and you'll end up with `'\''` which is a different string.